airflow/utils/file.py

```diff
@@ -388,7 +388,7 @@ def iter_airflow_imports(file_path: str) -> Generator[str, None, None]:
     """Find Airflow modules imported in the given file."""
     try:
         parsed = ast.parse(Path(file_path).read_bytes())
-    except (OSError, SyntaxError, UnicodeDecodeError):
+    except (OSError, SyntaxError, UnicodeDecodeError, ValueError):
```

Comment:
How about just `Exception` ?

Reply:
I'm not opposed to that, it is getting a little long :)